Errors building nlohmann_json_ext

For some reason, trying to build the nlohmann json extension module breaks with the following:

```
/Users/gruber/Projects/github.com/mcpe/mcpelauncher/build/nlohmann_json_ext-prefix/src/nlohmann_json_ext/nlohmann/detail/conversions/from_json.hpp:294:9: error: 
      static_assert failed "could not find from_json() method in T's namespace"
        static_assert(sizeof(BasicJsonType) == 0,
        ^             ~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Am I missing a critical dep or something?

--- a/ext/json.cmake
+++ b/ext/json.cmake
@@ -2,7 +2,7 @@ include(ExternalProject)
 
 ExternalProject_Add(
         nlohmann_json_ext
-        URL "https://github.com/nlohmann/json/releases/download/v3.1.2/include.zip"
+        URL "https://github.com/nlohmann/json/releases/download/v3.7.0/include.zip"
         CONFIGURE_COMMAND "" BUILD_COMMAND "" INSTALL_DIR "" INSTALL_COMMAND ""
 )
 ExternalProject_Get_Property(nlohmann_json_ext SOURCE_DIR)

